.github/workflows/build_and_test.yml

```diff
@@ -105,7 +105,7 @@ jobs:
           GH_REF: ${{ github.ref }}
           SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
         run: |
-          if [ "$GH_EVENT" = pull_request ]; then
+          if [ "$GH_EVENT" = pull_request ] && [ "${{ github.event.pull_request.head.repo.fork }}" != "true" ]; then
```

Comment:
suggestion: 

Instead of add the verification here, where it will aways run this pipeline, we could add a verification at step level and it would skip this pipeline and no change would be required here. Something like we did to send or not the slack notification [here](https://github.com/rsksmart/rskj/pull/2774/files#diff-8d30e942125e8823a02e5f4dd034f43550be2f67e40291d4f554623af93346cbL141).

Add this line before the `env` in the line 100:

`        if: github.event.pull_request.head.repo.fork != 'true'`

Reply:
Got it. Well, if that's the case you can let as it is.